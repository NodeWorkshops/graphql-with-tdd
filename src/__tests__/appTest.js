describe("Example test", () => {
    test("It should pass", async () => {
        expect(1).toBe(1);
    })
})