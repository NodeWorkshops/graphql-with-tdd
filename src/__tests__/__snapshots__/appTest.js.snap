// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Graphql status It should respond with a success message 1`] = `
Object {
  "data": Object {
    "status": "GraphQL status: OK",
  },
}
`;
